xml/System.Composition.Convention/AttributedModelProvider.xml

```diff
@@ -28,8 +28,8 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Called from constructors in derived classes to initialize the <see cref="AttributedModelProvider"/> class.</summary>
```

Comment:
Generally, we just want to document protected constructors as, "Creates a new instance of the <whatever> class."
In the Remarks, you can add a comment that this is a protected constructor that can only be called from derived classes.